src/core/webview/ClineProvider.ts

```diff
@@ -2074,22 +2075,24 @@ export class ClineProvider extends EventEmitter<ClineProviderEvents> implements
 						}
 						break
 
-					case "fetchMcpMarketplace": { // kilocode_change
+					// kilocode_change
```

Comment:
this one has an end!
```suggestion
					// kilocode_change_start
```